src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -36,6 +36,151 @@ the MIT License. See LICENSE in the project root for license information. -->
             <local:HasNestedCommandsVisibilityConverter x:Key="HasNestedCommandsVisibilityConverter"/>
             <model:IconPathConverter x:Key="IconSourceConverter"/>
 
+            <DataTemplate x:Key="GeneralItemTemplate" x:DataType="local:FilteredCommand">
+
+                <!-- This HorizontalContentAlignment="Stretch" is important
+                        to make sure it takes the entire width of the line -->
+                <ListViewItem HorizontalContentAlignment="Stretch"
+                                  IsTabStop="False"
+                                  AutomationProperties.Name="{x:Bind Item.Name, Mode=OneWay}"
+                                  AutomationProperties.AcceleratorKey="{x:Bind Item.KeyChordText, Mode=OneWay}">
+
+                    <Grid HorizontalAlignment="Stretch" ColumnSpacing="8" >
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="16"/>
+                            <!-- icon -->
+                            <ColumnDefinition Width="Auto"/>
+                            <!-- command label -->
+                            <ColumnDefinition Width="*"/>
+                            <!-- key chord -->
+                            <ColumnDefinition Width="16"/>
+                            <!-- gutter for scrollbar -->
+                        </Grid.ColumnDefinitions>
+
+                        <IconSourceElement
+                                    Grid.Column="0"
+                                    Width="16"
+                                    Height="16"
+                                    IconSource="{x:Bind Item.Icon,
+                                                        Mode=OneWay,
+                                                        Converter={StaticResource IconSourceConverter}}"/>
+
+                        <local:HighlightedTextControl
+                                    Grid.Column="1"
+                                    HorizontalAlignment="Left"
+                                    Text="{x:Bind HighlightedName, Mode=OneWay}"/>
+
+                        <!-- The block for the key chord is only visible
+                                when there's actual text set as the label. See
+                                CommandKeyChordVisibilityConverter for details. -->
+                        <Border
+                                    Grid.Column="2"
+                                    Visibility="{x:Bind Item.KeyChordText,
+                                                 Mode=OneWay,
+                                                 Converter={StaticResource CommandKeyChordVisibilityConverter}}"
+                                    Style="{ThemeResource KeyChordBorderStyle}"
+                                    Padding="2,0,2,0"
+                                    HorizontalAlignment="Right"
+                                    VerticalAlignment="Center">
```

Comment:
Since we're now duplicating a good amount of styling, maybe define a style at the top of the file and use it on both? For this one in particular, you could create a style `BasedOn={ThemeResource KeyChordBorderStyle}`.

Reply:
@carlos-zamora - can you please elaborate? :blush: I thought here I only use a style, while `BasedOn` is required when defining a new style. I am not strong at these things :blush:

Reply:
My bad. I thought both of the data templates were pretty similar. But there actually doesn't seem to be much overlap. If there was more overlap, I was hoping you could create a new style at the top, base it on `KeyChordBorderStyle`, and use the new style throughout both data templates.